packages/botonic-plugin-flow-builder/src/content-fields/hubtype-fields/queue-status-conditional.ts

```diff
@@ -0,0 +1,53 @@
+import { HtBaseNode, HtNodeLink } from './common'
+import { HtFunctionArguments, HtFunctionResult } from './function'
+import { HtNodeWithContentType, HtNodeWithoutContentType } from './node-types'
+
+// interface HtQueueArgument {
+//   locale: string
+//   values: [
+//     {
+//       type: 'string'
+//       name: 'queue_id'
+//       value: string
+//     },
+//     {
+//       type: 'string'
+//       name: 'queue_name'
+//       value: string
+//     },
+//   ]
+// }
+
+// interface HtCheckAvailableAgentsArgument {
+//   type: 'boolean'
+//   name: 'check_available_agents'
+//   value: boolean
+// }
+
+// export interface HtQueueStatusConditionalArguments {
+//   values: HtQueueArgument[] | HtCheckAvailableAgentsArgument[]
+// }
```

Comment:
missing?

Reply:
Related with the other comments. These is the exact types for conditional queue status